2025 on landing, clarification native workers to 0, more details on tags (#798)
windmill-labs/windmilldocs: docs/advanced/18_instance_settings/index.mdx

## Telemetry

> Anonymous usage data is collected to help improve Windmill. It's a simple HTTP to https://stats.windmill.dev. Stats are sent once a day. Stats only contain size of workers, their uptime and number of seats used.

Reviewer: ```suggestion
Anonymous usage data is collected to help improve Windmill. Telemetry data is sent as a HTTPS request to https://stats.windmill.dev. Stats are sent once a day. Stats only contain the size of workers, their uptime and number of seats used.
```
